packages/block-library/src/post-title/block.json

```diff
@@ -24,7 +24,7 @@
 		},
 		"linkTarget": {
 			"type": "string",
-			"default": "_blank"
+			"default": "_self"
```

Comment:
I've made this change in the other PR you reviewed after your comment (https://github.com/WordPress/gutenberg/pull/25714)

Reply:
Ah right. I just grep'd all Gutenberg for `_blank` and reviewed them, missed the fact it's on the other PR :+1: 
Removed it from this one since it will be included in #25714